Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -401,7 +408,16 @@ static inline void GeneratePixelShader(T& out, DSTALPHA_MODE dstAlphaMode, API_T
 
 		// On GLSL, input variables must not be assigned to.
 		// This is why we declare these variables locally instead.
-		out.Write("\tfloat4 col0, col1;\n");
+		if (ApiType == API_OPENGL)
```

Comment:
Might as well remove the API check here, I misread it in my PR, but make sure to init the vars for D3D =  float4(0.0, 0.0, 0.0 ,0.0); is best?
Not sure if OGL uses scalar promotion for = 0.0